developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst

```diff
@@ -2,7 +2,7 @@ System install on Ubuntu 16.04
 ==============================
 
 Here is a quick guide to setup the :doc:`system_requirements` on Ubuntu 16.04. This guide will help you to install all
-the packages and modules needed for Akeneo PIM, then configure it to match your local installation.
+the packages and modules needed for Akeneo PIM on a newly installed ubuntu 16.04 system, then configure it to match your local installation.
```

Comment:
Would you see this guide helpful in all situations which could occur when installing akeneo on a system with unknown usage history?